### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'window.screen.orientation.angle')

### ISSUE THREAD

**Issue by jmesserli** [outcobra/outstanding-cobra#178]
https://sentry.pegnu.cloud/pg/outstanding-cobra/issues/49/

```
TypeError: undefined is not an object (evaluating 'window.screen.orientation.angle')
  at None (/main.183f6c31b4253f5751fb.bundle.js:13:16633)
  at _next (/vendor.b63e0336aaa5b04105a7.bundle.js:1352:8124)
  at next (/vendor.b63e0336aaa5b04105a7.bundle.js:1104:5612)
  at debouncedNext (/vendor.b63e0336aaa5b04105a7.bundle.js:964:2193)
  at i (/vendor.b63e0336aaa5b04105a7.bundle.js:964:1282)
...
(9 additional frame(s) were not displayed)
```

**Comment by jmesserli** [comment]
(Mobile Safari 10 / iOS 10.1.1)

**Comment by M4R1KU** [comment]
New issue bc this is not resolved #257
